stripe/src/main/java/com/stripe/android/PaymentConfiguration.java

```diff
@@ -61,6 +62,15 @@ public PaymentConfiguration setShouldUseSourcesForCards(boolean shouldUseSources
         return this;
     }
 
+    void setEphemeralKeyProviderClassLoader(@NonNull ClassLoader classLoader) {
```

Comment:
This is actually part of my fud on calling `PaymentConfiguration` by that name -- it's really a `StripeConfiguration`, since it contains things that are useful beyond the `PaymentSession`